### DISCUSSION TITLE
Client not update error and no connection to clients

### DISCUSSION THREAD

**Discussion by FuchsEDV-MC** [amidaware/tacticalrmm#1349] [General]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04.5
 - Browser: chrome,Firefox,Edge
 - RMM Version (as shown in top left of web UI): v0.15.0

**Installation Method:**
  - [x] Standard

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): most of them are v2.0.4
- Agent OS: All types of Windows (10,11,SRV2012R2-2022)

**Describe the bug**
On 06.September 2022 at 20:30 all of our 474 Agents went offline and wont came back online.
There is no response since then. 
When checking the Tactical RMM Software on the clients they say "Agent: running, Mesh Agent: running".
But:
- Changes like Hostname or new Software on the client won't change in Tactical
- Agent Update not working
- Patch Management not working
- Run Script not working
- Software Install over Tactical not working

Still Working:
Old Harddrive Checks are working - we receive the notifications
Take Control (Mesh Agent) is working 

When we manually update a client or intall a new one then its working perfectly

It there a way to update all old (v.2.0.4 or older) clients automaticaly?
Which kind of tests could we perform to pinpoint and eliminate the fault?

**Comment by wh1te909** [comment]
start with reading the docs troubleshooting section and post your findings https://docs.tacticalrmm.com/troubleshooting/

**Reply by wh1te909** [reply]
also please read the 0.14.0 [release notes](https://github.com/amidaware/tacticalrmm/releases/tag/v0.14.0) it sounds like you closed port 4222 before allowing your agents to update (agent v2.0.4 was the last version to use 4222 before the move to websockets)
